### ISSUE TITLE
Display Tatsächliche Nutzung (actual use) in FE

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#158]
## Description
See Epic: #156 

## Depends on
#157  

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Corgam** [comment]
Already finished in one of the previous sprints, also including good performance for it.
